sklearn/utils/testing.py

```diff
@@ -747,6 +735,9 @@ def run_test(*args, **kwargs):
                                       reason='not compatible with PyPy')
     skip_if_no_parallel = pytest.mark.skipif(not joblib.parallel.mp,
                                              reason="joblib is in serial mode")
+    skip_if_no_matplotlib = pytest.mark.skipif(
```

Comment:
Do we still need this now that we have the `pyplot` fixture?

If it's only useful for the `ensemble.test_partial_dependence.py` tests, we should get rid of it.

Reply:
Let's remove it and introduce it whenever we will need it.